xray/commands/audit/generic/auditmanager.go

```diff
@@ -26,22 +26,24 @@ import (
 )
 
 type Params struct {
-	xrayGraphScanParams services.XrayGraphScanParams
-	serverDetails       *config.ServerDetails
-	progress            ioUtils.ProgressMgr
-	dependencyTrees     []*services.GraphNode
-	ignoreConfigFile    bool
-	excludeTestDeps     bool
-	insecureTls         bool
-	useWrapper          bool
-	depsRepo            string
-	releasesRepo        string
-	requirementsFile    string
-	technologies        []string
-	workingDirs         []string
-	args                []string
-	installFunc         func(tech string) error
-	xrayVersion         string
+	xrayGraphScanParams  *services.XrayGraphScanParams
+	serverDetails        *config.ServerDetails
+	progress             ioUtils.ProgressMgr
+	dependencyTrees      []*services.GraphNode
+	ignoreConfigFile     bool
+	excludeTestDeps      bool
+	insecureTls          bool
+	useWrapper           bool
+	withFixVersionFilter bool
+	minSeverityFilter    string
```

Comment:
minSeverityFilter --> filterBelowMinSeverity
Let's also change the related functions and variables names.

Let's also add a description to this param, describing its allowed values.